OpenEdXMobile/res/layout/discussion_response_action_bar_layout.xml

```diff
@@ -72,4 +72,8 @@
             style="@style/discussion_responses_small_text"
             android:text="@string/discussion_responses_report_label" />
     </LinearLayout>
+
+    <View
+        style="@style/light_gray_separator"
+        android:layout_gravity="top" />
```

Comment:
Overlapping in this case would be negligible and adding a `LinearLayout` specifically for this purpose seems to be an overkill. After all its 1dp and the buttons don't have a border defined.